Added new lock icon and google auth one

kxalex opened the pull request:
# Summary

The old hourglass icon did not fit the purpose of this plugin.

I have replaced it with a lock icon ![image](https://user-images.githubusercontent.com/284411/111756260-20ded300-88ee-11eb-9f71-5219cfc184a3.png) 36x36 (144DPI). 


Added bonus Google Authenticator icon ![image](https://user-images.githubusercontent.com/284411/111756220-11f82080-88ee-11eb-81a7-16ba150b3462.png) that can be used instead of default one.

![image](https://i.imgur.com/1AYh44b.png)

matryer commented:
@kxalex thank you for this. Could you please:

1. change `bitbar` metadata to `xbar`?
2. add an `xbar.image` metadata showing a screenshot of the plugin?

matryer commented:
@kxalex I'd love to get this merge if you could update the metadata?

kxalex commented:
> @kxalex I'd love to get this merge if you could update the metadata?

Just pushed a minor fix. Replaced `image` with `templateImage`, so it looks nice in light and dark modes.

Added image.
